### DISCUSSION TITLE
seed in json

### DISCUSSION THREAD

**Discussion by zjd11** [deepmodeling/deepmd-kit#993] [Q&A]
Hello,
I find there are three seed in input json fole, one in model section, one in fitting_net section , and one in training section.
what's the difference.
We plan to train four different model potentials on the same data set as DPgen did, which seed we need to change, or should all three be changed? thanks

**Comment by njzjz** [comment]
Three seeds controls trainable parameter initialization of the embedding network, the fitting network, and the order of training data, respectively. Changing any one may lead to a different result.

**Reply by zjd11** [reply]
Thanks
